forgettable-frontend/src/components/EncounterCardSummary/EncounterSummary.css

```diff
@@ -0,0 +1,58 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+@import  '../../colours.css';
+
+ .container{
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+}
+.card{
+    border-radius: 13;
+    height: 220px;
+    width:245px;
+}
+.header{
+    display: flex;
+    flex-direction: row;
+    padding-right:20px;
+    padding-left:20px;
+    padding-top: 20px;
+    font-weight:500;
+    font-size: 20px;
+    font-family:'Poppins', sans-serif;
+    line-height: 30px;
+    color: var(--ltxt1);
+}
+.description{
+    font-family: 'Poppins', sans-serif;
+    font-size: 13px;
+    font-weight: 400;
+    line-height: 21px;
+    color: var(--litext2);
+    font-style: normal;
+}
+.nameDesc{
+    padding-left: 8px;
+}
+.date{
+    display: flex;
+    flex-direction: row;
+    padding-left: 20px;
+    padding-right:20px;
+    padding-top:10px;
+    font-size: 13px;
+    font-family: 'Poppins', sans-serif;
```

Comment:
We no longer need to add font-family as I changed the default font family to Poppins :)